dev-python/numpy/numpy-1.19.4.recipe

```diff
@@ -56,13 +56,14 @@ eval "PROVIDES_${pythonPackage}=\"\
 	\"; \
 REQUIRES_$pythonPackage=\"\
 	haiku$secondaryArchSuffix\n\
-	numpy$secondaryArchSuffix\n\
+	cmd:cython$pythonVersion\n\
```

Comment:
The cython (for python 3.6 and 3.7) packages for 32 bit are secondary architecture so (same for BUILD_PREREQUIRES):
```diff
-	cmd:cython$pythonVersion\n\
+	cmd:cython$pythonVersion$secondaryArchSuffix\n\
```